package.json

```diff
@@ -36,12 +36,14 @@
     "apollo-server-lambda": "^1.3.4",
     "aws-lambda": "^0.1.2",
     "bluebird": "^3.5.1",
+    "dotenv": "^5.0.1",
     "graphql": "0.13.2",
     "graphql-playground-middleware-lambda": "^1.5.1",
     "graphql-tools": "2.24.0",
     "merge-graphql-schemas": "1.5.1",
     "moment": "^2.20.1",
     "mongoose": "^5.0.12",
+    "serverless": "^1.26.1",
```

Comment:
Why did you add this here? Edit: I like this, you are assuring we have a compatible version of the serverless framework.